client/src/modules/roles/modal/roleActions.html

```diff
@@ -0,0 +1,33 @@
+<div class="modal-header">
+    <ol class="headercrumb">
+      <li class="static" translate>FORM.BUTTONS.ACTIONS</li>
+      <li class="title">
+        <span translate>{{RoleActionsCtrl.role.label}}</span>
+      </li>
+    </ol>
+</div>
+<div class="modal-body">
+
+  <div class="row">
+    <div class="col-lg-12">
+      <ul class="list-group">
+        <li  class="list-group-item" ng-repeat="action in RoleActionsCtrl.actions">
+          <label class="radio-inline">
+            <input type='checkbox' ng-model='action.affected' ng-true-value="1" ng-false-value="0" />
+            <span id="{{action.id}}" translate>{{action.description}}</span>
```

Comment:
To avoid collisions with other element ids (since ids are supposed to be unique per page), why don't you add a prefix here?  Something like `id="action-{{action.id}}"`.  That way it will render as `id="action-1"` instead of `id="1"`.